src/JsonSchema/Constraints/Factory.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+/*
+ * This file is part of the JsonSchema package.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace JsonSchema\Constraints;
+
+use JsonSchema\Exception\InvalidArgumentException;
+use JsonSchema\Uri\UriRetriever;
+use JsonSchema\Validator;
+
+/**
+ * Factory for centralize constraint initialization.
+ */
+class Factory
```

Comment:
Well there are lot of opinions and this could be a large discussion.

1. #136 It's a workaround for a technical issue.
2. Recently a PHP 7 RFC has been approved relaxing reserved words based in context.
3. The purpose of namespaces is IMO to have clean class names.